tests/scripts/check-test-cases.py

```diff
@@ -26,6 +26,8 @@
 import sys
 
 class Results:
+    """Stores file and line information about errors
```

Comment:
Nitpick: [PEP 257](https://www.python.org/dev/peps/pep-0257) says to write ”Store …“, not “Stores …”, and to end with a period. PEP 257 also says a one-line docstring “should really fit on one line”, and a multi-line docstring should start with a one-line summary.

Reply:
I've updated this.